user/serializers.py

```diff
@@ -41,9 +41,7 @@ def validate(self, attrs):
             if user:
                 if not user.is_active:
                     msg = _("User account is disabled.")
-                    raise serializers.ValidationError(
-                        msg, code="authorization"
-                    )
+                    raise serializers.ValidationError(msg, code="authorization")
```

Comment:
The error message here refers to 'username', but the serializer uses 'email' for authentication. Consider updating the message to 'Must include 'email' and 'password'.' to avoid confusion.